test/mock/core/host_api/host_api_mock.hpp

```diff
@@ -292,25 +292,25 @@ namespace kagome::host_api {
 
     MOCK_METHOD(void,
                 ext_offchain_local_storage_set_version_1,
-                (runtime::WasmI32, runtime::WasmSpan, runtime::WasmSpan),
+                (runtime::WasmU32, runtime::WasmSpan, runtime::WasmSpan),
                 (override));
 
     MOCK_METHOD(void,
                 ext_offchain_local_storage_clear_version_1,
-                (runtime::WasmI32, runtime::WasmSpan),
+                (runtime::WasmU32, runtime::WasmSpan),
                 (override));
 
-    MOCK_METHOD(runtime::WasmI8,
+    MOCK_METHOD(runtime::WasmU32,
```

Comment:
And here too
![image](https://user-images.githubusercontent.com/9251127/147483699-2633151f-998f-4cae-ad33-d56fa24709e3.png)